src/shared/components/oncoprint/tabularDownload.ts

```diff
@@ -174,13 +174,16 @@ export function getTabularDownloadData(
                     : geneticTrackDatum.disp_mrna;
                 if (
                     oncoprintData.MRNA[currentGeneName][id] == '' ||
-                    oncoprintData.CNA[currentGeneName][id] == current_value
+                    oncoprintData.MRNA[currentGeneName][id] == current_value
```

Comment:
try to always use === instead of == (no coercion)
it's stricter.  probably doesn't make a difference but just a good practice